classes/topmenu.inc

```diff
@@ -83,7 +83,7 @@
                   <a href="http://openseamap.org/index.php?id=kartendownload"><strong>Download</strong></a>
                 </li>
             <li>
-              <a href="https://downloadlayer--onlinechart-ol3.netlify.com" target="_blank">OpenCPN (Kap)</a>
+              <a href="https://alpha.openseamap.org/#/download/.*/2040575595?lat=27.1957&lon=-12.0683&zoom=3&layers=A00000011000&lang=en" target="_blank">OpenCPN (Kap)</a>
```

Comment:
```suggestion
              <a href="https://alpha.openseamap.org/#/download/.*/null?lat=35.0245&lon=-63.2636&zoom=3&layers=A10001011000&lang=en" target="_blank">OpenCPN (Kap)</a>
```

One map was selected with the other link.

Reply:
That makes sense, thank you. When I use

> https://alpha.openseamap.org/#/download/.*/null?lat=35.0245&lon=-63.2636&zoom=3&layers=A10001011000&lang=en

the server redirects to 

> https://alpha.openseamap.org/#/download/.*/NaN?lat=35.0245&lon=-63.2636&zoom=3&layers=A10001011000&lang=en

So not sure whether null or NaN is preferred but it works so agreed.